### PR TITLE
Added Angular Connect talks about Angular 2.

### PR DESCRIPTION
**Pull request by 2fdevs**
This is related to issue #28.

Maybe we can create a list for every conference, so I've added a list of "Angular Connect" talks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,19 @@ Http is available as an injectable class, with methods to perform http requests.
 * [Egghead.io - Angular 2](https://egghead.io/technologies/angular2)
 * [udemy - Introduction to Angular 2](https://www.udemy.com/introduction-to-angular2/)
 
+#### Angular Connect
```
Comment (rolandjitsu) [inline README.md]: How about a link from the table of contents to this?

**Comment by SekibOmazic** [comment]
Maybe just link to the AngularConnect Channel on YouTube with all conference videos?

https://www.youtube.com/channel/UCzrskTiT_ObAk3xBkVxMz5g

File: README.md (hunk 1 above)
Comment (2fdevs) [inline README.md]: Added :)

**Comment by rolandjitsu** [comment]
@SekibOmazic In my opinion, it's better to have the titles there, because AC has a lot more talks about things other than Angular 2.

**Pull request merged**